pkg/asset/installconfig/openstack/utils.go

```diff
@@ -3,6 +3,7 @@ package openstack
 import (
```

Comment:
> pkg/asset/installconfig/openstack/utils.go

I do not think it is wise to name a file utils.go, it can easily become a place to keep all random functions, try to name this file a little more descriptive.